README.md

```diff
@@ -37,11 +37,22 @@ $ php bin/magento setup:static-content:deploy
 
 ## Configuring the plugin
 
-Once the plugin has been installed you will need to configure it. Go to the store configuration in the admin section and navigate to the payment methods. Here you will be able to configure the Komoju plugin, with your Komoju account details.
+Once the plugin has been installed you will need to configure it. Go to the store configuration in the admin section and navigate to the payment methods. Here you will be able to configure the KOMOJU plugin, with your KOMOJU account details. Your account details can be found on the [KOMOJU merchant settings page](https://komoju.com/admin/merchant_settings):
 
-### Configuring the Komoju webhook 
+![KOMOJU dashboard](./docs/screenshots/Komoju-merchant-settings.png "KOMOJU dashboard")
 
-To ensure that the Magento plugin works correctly you will need to set up a webhook from your Komoju dashboard to the wordpress instance. To do this you will need to go to your [Webhook page on the Komoju dashboard](https://komoju.com/admin/webhooks) and click "New Webhook". The Webhook URL is at `/komoju/hostedpage/webhook` of your Magento website. If your Magento was `https://magento.komoju.com` then your Webhook URL would be `https://magento.komoju.com/komoju/hostedpage/webhook`. The Secret can be anything you want (as long as you remember it), but you must make sure the following events are ticked:
+When configuring the "API Settings" section of the plugin use the following values:
+
+"Komoju merchant ID" in the plugin is "Merchant UUID" in the Merchant Settings dashboard.
+"Secret Key from Komoju" in the plugin is "Secret Key" in the Merchant Settings dashboard.
```

Comment:
We should capitalize these here and also in the plugin source

Reply:
Yep good thinking. Honestly, I think I'll just change the fields in the plugin to be the exact same name as the fields in the Merchant Settings page, that way we can eliminate any confusion.

Reply:
Updated in ed659bd